WORKSPACE

```diff
@@ -0,0 +1,7 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
```

Comment:
as per bazel, it may be worth changing this to http_archive: https://docs.bazel.build/versions/master/be/workspace.html

Reply:
I'd like to leave this as `git_repository`, because the auto-generated tarballs created by GitHub don't have stable checksums. See https://github.com/tensorflow/tensorflow/issues/12979 and https://github.com/bazelbuild/bazel/issues/3737 for some examples of GitHub breaking builds by tweaking their tarball builder.

`http_repository` does work well for GitHub "release" tarballs uploaded by maintainers, such as the Protobuf releases. Googletest doesn't have any of those, though.